Automattic/wp-calypso
# Stats: Add checklist banner component.

The component shows the next task in the onboarding checklist.

See p5XAZ9-1wt-p2 for more details on the checklist.

**The banner shows up in only `stage` or `development` environment.**

## How to test

1. Create a new site.
2. Switch to the new site on WordPress.com and select _Stats_ menu on the left sidebar.
3. You should be able to see the banner between the tabs and the chart as follows:
![desktop](https://user-images.githubusercontent.com/212034/33106882-dba9e4c4-cf77-11e7-9230-bc47aa3e0a8c.png)
![mobile-banner2](https://user-images.githubusercontent.com/212034/33106886-e0766540-cf77-11e7-9607-1df4c56198c1.png)
4. When you complete all tasks(or match the `completed` prop with `total`), you will see the completed banner.
![completed-banner](https://user-images.githubusercontent.com/212034/33106957-32190628-cf78-11e7-9d94-04d37de2731e.png)
5. Each social icon button pops up a window to share your website on social media.
6. If you click on the close button at the top-right of the banner, the banner never shows up again. Alternatively, you can hide the banner by setting `localStorage.setItem( 'onboarding-checklist-banner-closed', true );`
